src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/exceptions/NpgBadRequestException.kt

```diff
@@ -0,0 +1,10 @@
+package it.pagopa.ecommerce.eventdispatcher.exceptions
+
+import java.util.*
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ResponseStatus
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+class NpgBadRequestException(transactionID: UUID, cause: String = "N/A") :
```

Comment:
pass here transactionId using TransactionId value object. Please make sure to log transaction id without dashes to simplify transaction search. This comment target all newly added exceptions